src/Attributes/Generic.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Consolidation\AnnotatedCommand\Attributes;
+
+use Consolidation\AnnotatedCommand\Attributes\AttributeInterface;
+use Consolidation\AnnotatedCommand\Parser\CommandInfo;
+
+class Generic implements AttributeInterface
+{
+    protected const NAME = 'annotation-name';
+
+    public static function handle(\ReflectionAttribute $attribute, CommandInfo $commandInfo)
+    {
+        $commandInfo->addAnnotation(static::NAME, null);
```

Comment:
This is an example attribute, not a general-purpose attribute, I suppose?

Reply:
Its a base class. Its used by https://github.com/drush-ops/drush/pull/4821/files#diff-2a2e5ff65cd12399a0f627fc3881b2445974d8e0717fcb4f0c894c09fe4681de, for example. I renamed to NoArgumentsBase and made it abstract. This class is not used by AC so it could easily move to Drush if preferred.